builder.go

```diff
@@ -172,8 +172,11 @@ type Stage struct {
 	Node     *parser.Node
 }
 
-func NewStages(node *parser.Node, b *Builder) Stages {
+func NewStages(node *parser.Node, b *Builder) (Stages, error) {
```

Comment:
@nalind @smarterclayton @bparees Any concern with adding the error as part of the return values here?  I know Buildah can and will adjust.  Does the NewStages() function get called in OpenShift or other places where it may need to be adjusted too once this lands?

Reply:
we do call it directly but we can deal w/ that when we vendor the change in...  I can't speak to whether this is the *right* place to return the error though.  I assume it's ultimately going to get bubbled up to return from BuildDockerfiles() ?